src/main/java/moduletwo10/framework/constants/StringСonstants.java

```diff
@@ -0,0 +1,26 @@
+package moduletwo10.framework.constants;
+
+public class StringСonstants {
+
+    private StringСonstants() {
+    }
+
+    public static final String VALID_LOGIN = "tteplyak0va";
+    public static final String WRONG_LOGIN = "kkk";
+    public static final String VALID_PASSWD = "tanya1986";
+    public static final String WRONG_PASSWD = "12345";
+    public static final String ERROR_PASSWD = "Неверный пароль";
+    public static final String ERROR_CLICKING_TAB = "Error clicking tab ";
+    public static final String FOLDER_NAME = "newFolder9" +
+            "";
+    public static final String ERROR_FOLDER_NOT_CREATED = "Error. New folder not created ";
+    public static final String ERROR_DOCUMENT_NOT_CREATED = "Error. New document not created ";
+    public static final String ERROR_SAVING_DATA = "Error saving data in a document";
+    public static final String ERROR_CLICKING_FOLDER = "Error clicking folder ";
+    public static final String ERROR_FOLDER_NOT_DELETED = "Error. Folder not deleted.";
+    public static final String ERROR_FOLDER_NOT_IN_BASKET = "Error. Folder not in basket.";
+    public static final String ERROR_BASKED_IS_NOT_CLEARED = "Error. The basket is not cleared.";
+    public static final String DOCUMENT_NAME = "hello";
+    public static final String DOCUMENT_TEXT = "Hello World !!!";
+    public static final String SAVE_IN_YANDEX = "Сохранено в Yandex";
```

Comment:
It would be better to store these strings in appropriate classes. In TAF with hundreds of tests, pages, etc it will become too complicated, hard to read, and to maintain.